integrationTests/multiShard/smartContract/scCallingSC_test.go

```diff
@@ -428,9 +428,9 @@ func TestSCCallingDNSUserNames(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
-	numOfShards := 2
-	nodesPerShard := 2
-	numMetachainNodes := 2
+	numOfShards := 1
```

Comment:
this is in multishard package, might revert the values

Reply:
there is still one limitation - no breaking changes. patch will come later in the weekend.

Reply:
ok